src/web/mjs/connectors/templates/SpeedBinb.mjs

```diff
@@ -88,15 +88,15 @@ export default class SpeedBinb extends Connector {
     /**
      *************************
      *** SpeedBinb v01.6061 ***
-     * ** Comic Meteor       ***
+     * ** Comic Meteor, Digital Margaret ***
      *************************
      */
 
     /**
      *
      */
     _getPageList_v016061( imageConfigurtions, baseURL ) {
-        let pageLinks = imageConfigurtions.map( element => this.createConnectorURI( this.getAbsolutePath( element.dataset.ptimg, baseURL ) ) );
```

Comment:
I dont know what this modification implies, and since its on a template i'll ask other reviewers.